### ISSUE TITLE
max_memory_limit is not used

### ISSUE THREAD

**Issue by ChARCOOL** [VeXell/pm2-autoscale#2]
<img width="1724" alt="Screenshot 2024-02-06 at 23 10 35" src="https://github.com/VeXell/pm2-autoscale/assets/31819474/a062f8e4-25ca-49aa-8be6-cdcc2a1d88c8">

max_memory_limit is not used from the config ecoststem.config.js and console --max-memory-restart

**Comment by VeXell** [comment]
@ChARCOOL Hello.

I have `max_memory_limit` configuration in the plugin `package.json`. I guess the problem that plugin exceeds memory limit. I will increase it. It should not use  `max_memory_limit` from your `ecoststem.config.js`.

btw, could you provide screenshot of `pm2 status` command? 

```
"apps": [
        {
            "merge_logs": true,
            "max_memory_restart": "100M",
            "script": "index.js"
        }
    ]
```

**Comment by VeXell** [comment]
@ChARCOOL Published new npm package`1.2.1` please have a look on this build.
